<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eleve</class>
 <widget class="QDialog" name="eleve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="idE">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="genererbtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Generer</string>
   </property>
  </widget>
  <widget class="QLabel" name="background_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>451</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/blog_02.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="prenomE">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="editbtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Coordonn√©s de l'eleve:</string>
   </property>
  </widget>
  <widget class="QLabel" name="background_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nom:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletebtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="nomE">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="background_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Classe:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addbtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="background_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Prenom:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="classeE">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clearbtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="background_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Numero:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="numE">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>rechercher</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Eleve </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Inscription</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>idE</zorder>
  <zorder>genererbtn</zorder>
  <zorder>background_2</zorder>
  <zorder>prenomE</zorder>
  <zorder>editbtn</zorder>
  <zorder>label</zorder>
  <zorder>background_3</zorder>
  <zorder>deletebtn</zorder>
  <zorder>line</zorder>
  <zorder>nomE</zorder>
  <zorder>background_4</zorder>
  <zorder>addbtn</zorder>
  <zorder>background_5</zorder>
  <zorder>classeE</zorder>
  <zorder>clearbtn</zorder>
  <zorder>background_6</zorder>
  <zorder>numE</zorder>
  <zorder>pushButton</zorder>
  <zorder>lineEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
